backend/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import dotenv from "dotenv";
+import express from "express";
+import cors from "cors";
+import "reflect-metadata";
+import {createConnection} from "typeorm";
+import config from "./config/db.config";
+
+dotenv.config();
+
+const corsOptions = {
+    origin: process.env.CORS_ORIGIN || "*",
+};
+
+const PORT: Number = Number(process.env.PORT) || 8001;
+
+
+createConnection(config).then(async connection => {
```

Comment:
I'd promisify all callbacks
apart from that lgtm

Reply:
I didn't get it - what should I change exactly?